tools/topology/topology2/development/sof-plugin.conf

```diff
@@ -0,0 +1,170 @@
+<searchdir:include>
+<searchdir:include/common>
+<searchdir:include/components>
+<searchdir:include/dais>
+<searchdir:include/pipelines/cavs>
+<searchdir:platform>
+<searchdir:platform/intel>
+
+<vendor-token.conf>
+<manifest.conf>
+<tokens.conf>
+<host-gateway-capture.conf>
+<host-gateway-playback.conf>
+<io-gateway-capture.conf>
+<io-gateway.conf>
+<data.conf>
+<pcm.conf>
+<pcm_caps.conf>
+<fe_dai.conf>
+<ssp.conf>
+<route.conf>
+<intel/common_definitions.conf>
+<dai-copier.conf>
+<module-copier.conf>
+<pipeline.conf>
+<input_audio_format.conf>
+<output_audio_format.conf>
+
+Object.Pipeline.io-gateway-capture [
+	{
+		index		10
+		direction	capture
+
+		Object.Widget.dai-copier."1" {
+			dai_index	1
+			dai_type	"SSP"
```

Comment:
does the concept of SSP make any sense for a plugin at the ALSA-lib level? Why is this needed?

Reply:
it is not really needed for the plugin but these are mandatory attributes for a dai-copier and I cant create one without it. I will have to update the host/dai copier to remove these  but for now, we have no choice